clients/imodels-client-management/src/base/types/apiEntities/OperationInterfaces.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** Possible Create iModel Operation states. */
+export enum IModelCreationState {
+  /** iModel creation process completed successfully. */
+  Successful = "successful",
+  /** iModel is being created from a Baseline File and the file upload to file storage has not been completed yet. */
+  WaitingForFile = "waitingForFile",
+  /** iModel creation process is scheduled or in progress. */
+  Scheduled = "scheduled",
+  /** iModel creation process failed. */
+  Failed = "failed"
+}
+
+/** Information about the source iModel of an iModel clone. */
+export interface ClonedFrom {
+  /** Id of the source iModel. */
+  iModelId: string;
+  /**
+   * Id of the latest source iModel Changeset which was copied to this iModel.
+   * This corresponds to the Changeset specified in changesetId or changesetIndex properties when cloning an iModel.
```

Comment:
I think we surround property names with `` in jsdoc comments, at least most of the time

Reply:
updated